### ISSUE TITLE
Quarkus CLI: Create extension with Gradle as build tool

### ISSUE THREAD

**Issue by HerrDerb** [quarkusio/quarkus#30260]
### Description

Currently the command `quarkus create extension` is only able to generate maven templates.

Other sub commands, as `quarkus create app` and `quarkus create cli`  allow to create a new app template with gradle by adding the parameter `--gradle`

It would be desired, if `quarkus create extension` whould be aligned with the other sub commands, resulting in the option to create an extension template with gradle.

**Comment by maxandersen** [comment]
I believe we don't yet have a clean way to build extensions with gradle and that's why. 

@aloubyansky can you confirm/deny? :)

**Comment by aloubyansky** [comment]
True, it shouldn't be difficult to add that though. For now @HerrDerb please follow https://quarkus.io/guides/building-my-first-extension#gradle-setup

**Comment by glefloch** [comment]
@HerrDerb there are example in tests: https://github.com/quarkusio/quarkus/tree/main/integration-tests/gradle/src/main/resources/extensions/simple-extension

**Comment by HerrDerb** [comment]
@aloubyansky Thank you :)  I'm already on my second gradle extension, so the manual setup with gradle is technically not an issue, but it's time consuming and errorenous. Therefor, I would love to be able to just generate a basic extension project with gradle as build tool, same as with an app or cli project.

**Comment by aloubyansky** [comment]
Great to hear that @HerrDerb out of curiosity, what kind of extensions have you been working on? Anything the Quarkus community could find generally useful?

**Comment by HerrDerb** [comment]
@aloubyansky I am afraid not, the current extensions I am working on are very company specific, e.g for company specific needs (configuration, functionality) when using an existing extension in a multitenant setup.

**Comment by aloubyansky** [comment]
Good to know anyway and thanks for sharing @HerrDerb

**Comment by holly-cummins** [comment]
Is this a duplicate of https://github.com/quarkusio/quarkus/issues/30260, @HerrDerb?

**Comment by HerrDerb** [comment]
> Is this a duplicate of https://github.com/quarkusio/quarkus/issues/30260, @HerrDerb?

This is #30260 or am I seeing something wrong? 🤔

**Comment by holly-cummins** [comment]
> > Is this a duplicate of [#30260](https://github.com/quarkusio/quarkus/issues/30260), [@HerrDerb](https://github.com/HerrDerb)?
> 
> This is [#30260](https://github.com/quarkusio/quarkus/issues/30260) or am I seeing something wrong? 🤔

Oh dear. Now I'm going to have to find the other issue again somewhere in my tabs.

Ah, this is the one I meant, sorry, @HerrDerb! https://github.com/quarkusio/quarkus/issues/3322

**Comment by HerrDerb** [comment]
😄 thanks.  Yes this seems tolle about the same👍
